app/models/automated_checks/claim_verifiers/provider_verification.rb

```diff
@@ -36,9 +36,89 @@ def create_task!
       end
 
       def passed?
-        verification.fetch("assertions").all? do |assertion|
-          assertion.fetch("outcome") == true
+        # Year 2 auto-pass logic for FE eligibility task
+        # Task passes automatically when ALL conditions are met:
+        # 1. Provider selects 'Yes' to: Teaching responsibilities, First 5 years of teaching, Age range taught, Subject, Course
+        # 2. Provider selects 'No' to: Subject to performance measures, Subject to disciplinary action
+        # 3. Provider and Claimant responses match for: Contract of employment, Timetabled teaching hours
+        # 4. Provider selects acceptable teaching qualification (Yes/Not yet enrolled/Planning to enrol)
+
+        provider_yes_required = %w[teaching_responsibilities in_first_five_years half_teaching_hours subjects_taught taught_at_least_one_term]  # Performance measures and disciplinary action are handled differently
+
+        # Check provider Yes responses
+        provider_yes_pass = provider_yes_required.all? do |field|
+          assertion = verification.fetch("assertions").find { |a| a["name"] == field }
+          assertion && assertion.fetch("outcome") == true
+        end
+
+        # Check performance measures and disciplinary action (provider should select No/false)
+        performance_disciplinary_pass = check_performance_and_disciplinary_responses
+
+        # Check matching responses between claimant and provider
+        matching_responses_pass = check_matching_responses
+
+        # Check teaching qualification acceptance
+        teaching_qualification_pass = check_teaching_qualification
+
+        provider_yes_pass && performance_disciplinary_pass && matching_responses_pass && teaching_qualification_pass
+      end
```

Comment:
I don't think this code will be run. The claim verifier isn't being called after the provider verification form is submitted.
On master we've removed this claim verifier (can always add it back), but the claim verifiers run on initial claim submission so would need to explicitly call this verifier after the provider has submitted the verification.